examples/test_client.py

```diff
@@ -0,0 +1,456 @@
+#!/usr/bin/env python
```

Comment:
Including it was a bit sloppy. I've since removed it in favour of putting formal tests in (TBD).